tests/Unit/classes/ValidateCoreTest.php

```diff
@@ -213,7 +258,7 @@ public function isUnsignedFloatDataProvider()
             $this->trueFloatDataProvider(),
             array(
                 array(false, -12.2151),
-                array(false, -12,2151),
+                array(false, -12, 2151),
```

Comment:
This test is weird

Reply:
Was an old test, cs-fixer just add a space here :D

Reply:
But yes, it's weird :/